### ISSUE TITLE
Chracter encoding issues

### ISSUE THREAD

**Issue by jmbanda** [OHDSI/Hermes#11]
After playing with this excellent and very useful tool, I have found an issue when looking at concepts with non-standard characters. This might be an encoding issue when rendering the contents of the database. The issue comes when looking for:

Ménière's disease

When I query for: Ménière we get 0 results, changing this to Meniere we do get results. When clicking on the SNOMED entry (direct link: http://ohdsi.org/web/hermes/index.html#/concept/439717) There is one entry that shows the issue (Active M??'s disease, cochleovestibular). I have attached a screen grab of this.

![hermes-encoding issue](https://cloud.githubusercontent.com/assets/10423944/5633442/2f9cc444-958c-11e4-9bbe-8530abc14119.PNG)

I have tested this on the latest Firefox using Windows, in case it might be an encoding issue on my side.

**Comment by fdefalco** [comment]
I'm seeing slightly different behavior across environments on this issue.  Our public version (postgres I believe) fails to return results with Ménière.  My local version using MS SQL finds it without issue.  It appears as if the public postgres version of the database doesn't include the special characters as I can find the same concept identifier identified concepts in both environments, one with the special accented characters, and one without.  

So, to summarize, when going to the same URL in the two different environments the concept is accessible however the special characters only exist in one (the ms sql environment).  I'm guessing this has something to do with how the vocab is loaded into postgres. 

I am finding the a single quote does break my local environment and I'll have to update it to replace with two single quotes, as that should resolve the issue across environments.

**Comment by schuemie** [comment]
One possible problem might also be that we haven't picked a character encoding standard. I've opened a [discussion](http://forums.ohdsi.org/t/character-encoding-standard/208/1) in our forums.

**Comment by jmbanda** [comment]
Indeed, I had the issue when loading to Postgres, However, after parsing the vocabulary files and removing some Windows special characters that the version I used had. I was able to load them correctly. 

In terms of character encoding standard, UMLS uses UTF-8 and this works very well for the very few concepts that include accents/special characters.
